src/index.ts

```diff
@@ -5,7 +5,7 @@ import { availableParallelism } from 'os';
 import { fileURLToPath, URL } from 'url';
 import { resolve } from 'path';
 import { inspect, types } from 'util';
-import assert from 'assert';
+import { strict as assert } from 'assert';
```

Comment:
I doubt the change to use `strict` is visible to consumers, but `assert` is used as a function, so we need to tweak this import